### PR TITLE
Remove dependency on localevent.h

### PR DESCRIPTION
**Pull request by kant2002**

### CODE REVIEW

File: src/fheroes2/game/fheroes2.cpp [hunk 1]
```diff
@@ -275,6 +275,17 @@ bool ReadConfigs( void )
                 if ( !externalCommand.empty() )
                     Music::SetExtCommand( externalCommand );
 
+                LocalEvent & le = LocalEvent::Get();
+                if ( conf.PocketPC() ) {
```
Comment (kant2002) [inline src/fheroes2/game/fheroes2.cpp:279]: Given that I found #1055, maybe better just drop that code? or PocketPC still supported.

Comment (ihhub) [reply src/fheroes2/game/fheroes2.cpp:279]: Yes, We must drop PocketPC code. We don't support it at all.

**Comment by kant2002** [comment]
@ihhub I will remove PocketPC code in subsequent PR.

**Comment by ihhub** [comment]
@kant2002 thank you so much for all these code cleanups!

**Comment by kant2002** [comment]
For the Horde!

File: src/fheroes2/system/settings.cpp [hunk 2]
```diff
@@ -662,25 +660,6 @@ bool Settings::Read( const std::string & filename )
     if ( !sval.empty() )
         video_driver = sval;
 
-    // pocketpc
-    if ( PocketPC() ) {
```
Comment (kant2002) [inline src/fheroes2/system/settings.cpp:666]: @ihhub I decide to do PocketPC cleanup and found that Android potentially can use same code paths as PocketPC under System::isEmbeddedDevice setting some PocketPC flags. It is late and I likely can miss something important, so I will take closer look at day, or so. But if you know some Android maintainer which can do a quick test that can also helps.

**Pull request merged**